dkg-gadget/src/utils.rs

```diff
@@ -31,11 +30,11 @@ pub fn validate_threshold(n: u16, t: u16) -> u16 {
 pub fn set_up_rounds<N: AtLeast32BitUnsigned + Copy>(
 	authority_set: &AuthoritySet<AuthorityId>,
 	public: &AuthorityId,
-	sr25519_public: &sr25519::Public,
+	_sr25519_public: &sr25519::Public,
```

Comment:
Maybe add `TODO:` above fn params we are not currently using?

Reply:
Alright